kevm-pyk/src/kevm_pyk/kproj/evm-semantics/schedule.md

```diff
@@ -1431,5 +1431,108 @@ A `ScheduleConst` is a constant determined by the fee schedule.
+    rule Glogtopic < PRAGUE > => 375
+
+    rule Gcall        < PRAGUE > => 700
+    rule Gcallstipend < PRAGUE > => 2300
+    rule Gcallvalue   < PRAGUE > => 9000
+    rule Gnewaccount  < PRAGUE > => 25000
+
+    rule Gcreate       < PRAGUE > => 32000
+    rule Gcodedeposit  < PRAGUE > => 200
+    rule Gselfdestruct < PRAGUE > => 5000
+    rule Rselfdestruct < PRAGUE > => 0
+
+    rule Gmemory      < PRAGUE > => 3
+    rule Gquadcoeff   < PRAGUE > => 512
+    rule Gcopy        < PRAGUE > => 3
+    rule Gquaddivisor < PRAGUE > => 3
+
+    rule Gtransaction   < PRAGUE > => 21000
+    rule Gtxcreate      < PRAGUE > => 53000
+    rule Gtxdatazero    < PRAGUE > => 4
+    rule Gtxdatanonzero < PRAGUE > => 16
+
+    rule Gjumpdest    < PRAGUE > => 1
+    rule Gbalance     < PRAGUE > => 700
+    rule Gblockhash   < PRAGUE > => 20
+    rule Gextcodesize < PRAGUE > => 700
+    rule Gextcodecopy < PRAGUE > => 700
+
+    rule Gecadd       < PRAGUE > => 150
+    rule Gecmul       < PRAGUE > => 6000
+    rule Gecpairconst < PRAGUE > => 45000
+    rule Gecpaircoeff < PRAGUE > => 34000
+    rule Gfround      < PRAGUE > => 1
+
+    rule maxCodeSize < PRAGUE > => 24576
+    rule Rb          < PRAGUE > => 0
+
+    rule Gcoldsload             < PRAGUE > => 2100
+    rule Gcoldaccountaccess     < PRAGUE > => 2600
+    rule Gwarmstorageread       < PRAGUE > => 100
+    rule Gwarmstoragedirtystore < DEFAULT > => 100
+
+    rule Gaccessliststoragekey < PRAGUE > => 1900
+    rule Gaccesslistaddress    < PRAGUE > => 2400
+
+    rule maxInitCodeSize   < PRAGUE > => 49152
+    rule Ginitcodewordcost < PRAGUE > => 2
+
+    rule Rmaxquotient < PRAGUE > => 5
+
+    rule Gselfdestructnewaccount << PRAGUE >> => true
+    rule Gstaticcalldepth        << PRAGUE >> => false
+    rule Gemptyisnonexistent     << PRAGUE >> => true
+    rule Gzerovaluenewaccountgas << PRAGUE >> => false
+    rule Ghasrevert              << PRAGUE >> => true
+    rule Ghasreturndata          << PRAGUE >> => true
+    rule Ghasstaticcall          << PRAGUE >> => true
+    rule Ghasshift               << PRAGUE >> => true
+    rule Ghasdirtysstore         << PRAGUE >> => true
+    rule Ghassstorestipend       << PRAGUE >> => true
+    rule Ghascreate2             << PRAGUE >> => true
+    rule Ghasextcodehash         << PRAGUE >> => true
+    rule Ghasselfbalance         << PRAGUE >> => true
+    rule Ghaschainid             << PRAGUE >> => true
+    rule Ghasaccesslist          << PRAGUE >> => true
+    rule Ghasbasefee             << PRAGUE >> => true
+    rule Ghasrejectedfirstbyte   << PRAGUE >> => true
+    rule Ghasprevrandao          << PRAGUE >> => true
+    rule Ghasmaxinitcodesize     << PRAGUE >> => true
+    rule Ghaspushzero            << PRAGUE >> => true
+    rule Ghaswarmcoinbase        << PRAGUE >> => true
+    rule Ghastransient           << PRAGUE >> => true
+    rule Ghasmcopy               << PRAGUE >> => true
+    rule Ghasbeaconroot          << PRAGUE >> => true
+
```

Comment:
It seems you are missing some blob rule at the end of the list. Can you double-check?

Reply:
Ah, sorry, I used an older version of the file as a template. Fixed.

Reply:
no problem! Thanks!